index.js

```diff
@@ -194,9 +207,12 @@ function getFullNames(/* CODE HERE */) {
  * @returns an array with all the runners' first names in ALL CAPS.
  * The first names appear in the array in the same order the runners appear in the `runners` array.
 */
-function firstNamesAllCaps(/* CODE HERE */) {
-  /* CODE HERE */
-}
+function firstNamesAllCaps(runners) {
+  const firstNames = runners.map(function(upperName){
+    return upperName.first_name.toUpperCase()
+  });
+  return firstNames;
+};
 
```

Comment:
Good job, try to use arrow functions, you will rely on them a lot in the future.